core/domain/stats_services.py

```diff
@@ -389,6 +383,46 @@ def update_exp_issues_for_new_exp_version(
     create_exp_issues_model(exp_issues)
 
 
+def get_exp_issues(exp_id, exp_version):
+    """Retrieves the ExplorationIssues domain object.
+
+    Args:
+        exp_id: str. ID of the exploration.
+        exp_version: int. Version of the exploration.
+
+    Returns:
+        ExplorationIssues|None: The domain object for exploration issues or
+            None if the exp_id is invalid.
+    """
+    exp_issues = None
+    exp_issues_model = stats_models.ExplorationIssuesModel.get_model(
+        exp_id, exp_version)
+    if exp_issues_model is not None:
+        exp_issues = get_exp_issues_from_model(exp_issues_model)
+    return exp_issues
+
+
+def get_playthrough_by_id(playthrough_id):
+    """Retrieves the Playthrough domain object.
+
+    Args:
+        playthrough_id: str. ID of the playthrough.
+
+    Returns:
+        Playthrough|None: The domain object for the playthrough or None if the
+            playthrough_id is invalid.
+
+    Raises:
+        Exception: Entity for class PlaythroughModel with id not found.
```

Comment:
It doesn't look like this is ever raised b/c of `strict=False`. Are you sure this is still correct?

Reply:
Correct Removed it